sonic-pcied/scripts/pcied

```diff
@@ -92,6 +96,7 @@ class DaemonPcied(daemon_base.DaemonBase):
         self.state_db = daemon_base.db_connect("STATE_DB")
         self.device_table = swsscommon.Table(self.state_db, PCIE_DEVICE_TABLE_NAME)
         self.status_table = swsscommon.Table(self.state_db, PCIE_STATUS_TABLE_NAME)
+        self.detach_info = swsscommon.Table(self.state_db, PCIE_DETACH_INFO_TABLE)
```

Comment:
@vvolam I think we can make use of existing DPU state when DPU is detached from PCI...lets check that

Reply:
@prgeor please check the reply in above comment.